test/AccessControl.js

```diff
@@ -0,0 +1,109 @@
+import { expect } from "chai";
+import EvmError from "./helpers/EVMThrow";
+import eventValue from "./helpers/eventValue";
+
+const RoleBasedAccessControl = artifacts.require("RoleBasedAccessControl");
+const TestAccessControlTruffleMixin = artifacts.require(
+  "TestAccessControlTruffleMixin"
+);
+
+contract("AccessControl", ([accessController, owner1, owner2]) => {
+  let accessControl;
+  let accessControlled;
+  let exampleRole;
+
+  beforeEach(async () => {
+    accessControl = await RoleBasedAccessControl.new();
+    accessControlled = await TestAccessControlTruffleMixin.new(
+      accessControl.address
+    );
+
+    exampleRole = await accessControlled.ROLE_EXAMPLE();
+  });
+
+  function expectAccessChangedEvent(
```

Comment:
This can be generalized to any event and put in helpers. But let's do that in a separate PR.